composer.json

```diff
@@ -12,14 +12,20 @@
         }
     ],
     "require": {
-        "php": "^5.4|^7.0"
+        "php": "^7.0",
+        "psr/http-client": "^1.0",
+        "psr/http-factory": "^1.0"
```

Comment:
none of the changes above here are legit: this is a HUGE change for the project, and unrelated to what we want to achieve (which is allowing other kind of clients next to existing ones)